samples/cuda_samples/CMakeLists.txt

```diff
@@ -132,10 +77,70 @@ add_cuda_sample(cuda-sobolqrng  4_Finance/SobolQRNG sobol.cpp  sobol_gold.cpp  s
 add_cuda_sample(cuda-reduction  6_Advanced/reduction reduction.cpp  reduction_kernel.cu)
 add_cuda_sample(cuda-fastwalsh  6_Advanced/fastWalshTransform fastWalshTransform.cu  fastWalshTransform_gold.cpp)
 
-# 6_Advanced/shfl_scan/shfl_integral_image.cuh:31:33: error: use of undeclared identifier 'warpSize'
-# 6_Advanced/shfl_scan/shfl_integral_image.cuh:83:17: error: no matching function for call to '__shfl_up'
-# error: no matching function for call to '__shfl_xor'
-#add_cuda_sample(cuda-shflscan  6_Advanced/shfl_scan shfl_scan.cu)
+############################
+
+if(NOT ENABLE_FAILING_TESTS)
+  ##### when run, sometimes fails sometimes passes TODO investigate
+  set_tests_properties("cuda-simpleCallback" PROPERTIES DISABLED 1 LABELS "")
+endif()
+
+if(ENABLE_UNCOMPILABLE_TESTS)
```

Comment:
Do we have any such tests?

Reply:
I have added them to `samples/cuda-samples`